cookbooks/bcpc/recipes/chef_vault_download.rb

```diff
@@ -0,0 +1,50 @@
+#
+# Cookbook Name:: bcpc
+# Recipe:: chef vault install
+#
+# Copyright 2015, Bloomberg Finance L.P.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+target_filename = "chef-vault-2.2.4.gem"
+link_filename = "chef-vault.gem"
+package_path = "#{node['bcpc']['bin_dir']['gems']}/#{target_filename}"
+checksum = "8d89c96554f614ec2a80ef20e98b0574c355a6ea119a30bd49aa9cfdcde15b4a"
+
+# download rubygem from website
+remote_file "#{ target_filename }" do
+  source "#{ node['bcpc']['gem_source'] }/#{ target_filename }"
+  checksum checksum
+  group "root"
+  owner "root"
+  mode "755"
+  path package_path
+  action :nothing
```

Comment:
I would think we want a guard here to not keep downloading the file (e.g. a cluster which does not have internet access but does have bins pre-stashed).